device.go

```diff
@@ -100,6 +100,16 @@ type DeviceInfo struct {
 	ID string
 }
 
+//Generic function for creating a named-id for passing on the hypervisor commandline
+func makeNameID(namedType string, id string) string {
```

Comment:
It is still in `device.go`. I thought we had agreed on moving this to `hypervisor.go`
Also, please update your commit message.